tests/functional/commands/test_setup_command.py

```diff
@@ -81,6 +83,36 @@ def test_stop_creation_when_exists(monkeypatch, tmp_path):
     assert content == dump
 
 
+def test_invalid_database(monkeypatch, tmp_path):
+    """Test if command prompt loops when invalid database is typed."""
+    config_path = str(tmp_path) + "/tmp_config.yaml"
+    monkeypatch.setattr(orion.core, "DEF_CONFIG_FILES_PATHS", [config_path])
+    monkeypatch.setattr(
+        builtins,
+        "input",
+        _mock_input(
+            [
+                "invalid database",
+                "invalid database again",
+                "2383ejdd",
+                "another invalid database",
+                "mongodb",
+                "the host",
+                "the name",
+            ]
+        ),
+    )
+
```

Comment:
Just to be safe I would use caplog and verify that `Unexpected value` with valid choices was printed at least once.

Reply:
Done using capsys !